src/line_range.rs

```diff
@@ -74,34 +92,45 @@ impl LineRange {
 #[test]
 fn test_parse_full() {
     let range = LineRange::from("40:50").expect("Shouldn't fail on test!");
```

Comment:
I think we need think more about the interface here. I would expect `LineRange::from("40:50")` to only return a single line range.